src/append_only_data.rs

```diff
@@ -1026,4 +1045,70 @@ mod tests {
         let decoded = unwrap!(self::Address::decode_from_zbase32(&encoded));
         assert_eq!(address, decoded);
     }
+
+    #[test]
+    fn append_unseq_data_test() {
+        let mut data = UnpubUnseqAppendOnlyData::new(XorName(rand::random()), 10);
+
+        // Assert that the current Append Operation has duplicate entries present and are not pushed
```

Comment:
Maybe rephrase it a bit? Also, we're looking for duplicate _keys_ as opposed to entries (keys + values).
```suggestion
        // Assert that the entries are not appended because of duplicate keys.
```